scripts/generate-groups.sh

```diff
@@ -61,7 +61,7 @@ shift 4
   # To support running this script from anywhere, we have to first cd into this directory
   # so we can install the tools.
   cd "$(dirname "${0}")"
-  go install k8s.io/code-generator/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}
+  go install k8s.io/code-generator/cmd/{defaulter-gen,client-gen,lister-gen,informer-gen,deepcopy-gen}@v0.27.1
```

Comment:
nit: Should this be 1.27.2 ? same as the go.mod file can be fixed on commit if easier

Reply:
Good catch. Original patch used 1.27.1 versions, but output is identical. I've updated locally, and will do so on commit as well.